docs/changes.rst

```diff
@@ -431,6 +431,11 @@ This version contains all fixes up to pywbem 0.12.4.
   This is only a workaround; An issue against python-coveralls has been
   opened: https://github.com/z4r/python-coveralls/issues/66
 
+* Added the concept of end2end tests for pywbem. These are tests against real
+  WBEM servers. The servers to test against can be specified in a WBEM server
```

Comment:
should mention that it allows users to define new tests to be executed by the end2end infrastructure.